Functions/xASL_quant_GetMeanControl.m

```diff
@@ -36,29 +36,36 @@
         
     % For example for 64 volumes and 2 repetitions with 8 PLDs and 4 TEs, it takes volume 1 and 33
     imMeanControl = imASLTimeSeries(:,:,:,1:blockSize:end);
-else
-	% Create mean control in native space (works also for multi-PLD)
-	imMeanControl = xASL_quant_GetControlLabelOrder(imASLTimeSeries);
 	
-	% For multi-PLD, we need to select one PLD
-	if isfield(x.modules.asl,'bMultiPLD') && x.modules.asl.bMultiPLD
+elseif isfield(x.modules.asl,'bMultiPLD') && x.modules.asl.bMultiPLD
+	% Create mean control in native space
+	if x.modules.asl.bDeltaM
```

Comment:
We should probably rename to `bContainsDeltaM`.
Yes, of course, it is automatically detected option. But we have the automatic detection centralized and then use this variable, instead of re-detecting at each point. 
Previously, this was wrong as it detected this as having a single volume - which is not with multi-PLD not an effective thing to do.

Reply:
Yes if you also add `bContainsLookLocker` ;)

Reply:
`bDeltaM` renamed to `bContainsDeltaM` 312b844 .Can you resolve?